README.md

```diff
@@ -1,17 +1,18 @@
-# JKAN 
-A lightweight, backend-free open data portal, powered by Jekyll
-
-Open-source data portals can be really hard to install and maintain. But their
-basic purpose of providing links to download data really isn't that complicated. JKAN is a proof-of-concept
-that allows a small, resource-strapped government agency to stand-up an open data portal by simply
-[clicking the fork button](https://help.github.com/articles/fork-a-repo/).
-
-[Demo site](https://demo.jkan.io)
-
-## Documentation
-* [Installation](docs/installation.md)
-* [Upgrading](docs/upgrading.md)
-* [Schema customization](docs/schema-customization.md)
-* [Running locally](docs/running-locally.md)
-* [Adding and editing content](docs/content.md)
-* [Architecture](docs/architecture.md)
+# OpenDataPhilly
+
+[OpenDataPhilly](https://www.opendataphilly.org/) is a regional open data catalog, providing a repository for information about open data available from the City of Philadelphia, non-profit organizations, university research projects, and other publishers of open data about the region.

```

Comment:
no www needed. nitpick but planning to eventually scan for and detect redirects so we can ensure links work